src/rocker/extensions.py

```diff
@@ -162,6 +162,26 @@ def register_arguments(parser, defaults={}):
             default=defaults.get('network', None),
             help="What network configuration to use.")
 
+class IPC(RockerExtension):
```

Comment:
Likewise stick this in alphabetical order since it's otherwise not otherwise constrained for easier reading.

Reply:
I see Expose is in the wrong place too, but we can converge on a better layout.